### PR TITLE
quick_sort/quick_sort.py: Add quick_sort in python3.

### PR DESCRIPTION
**Pull request by Naman9639**

### CODE REVIEW

File: quick_sort/quick_sort.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            i = i+1 
+            arr[i],arr[j] = arr[j],arr[i]
+    arr[i+1],arr[high] = arr[high],arr[i+1]
+    return ( i+1 )
+def quickSort(arr,low,high):
+    if low < high:        
+        pi = partition(arr,low,high)        
+        quickSort(arr, low, pi-1)
+        quickSort(arr, pi+1, high)
+arr = random.sample(range(10000),10)
```
Comment (t2013anurag) [inline quick_sort/quick_sort.py]: new line.

File: quick_sort/quick_sort.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+n = len(arr)
+x = time.time()
+quickSort(arr,0,n-1)
```
Comment (t2013anurag) [inline quick_sort/quick_sort.py]: new line.

File: quick_sort/quick_sort.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+y = time.time()
+print("Time taken by quick sort is: ")
+print(y-x)
+print ("Sorted array is:")
+for i in range(n):
```
Comment (t2013anurag) [inline quick_sort/quick_sort.py]: new line.

**Comment by t2013anurag** [comment]
@Naman9639  -  Also squash your commits into one.

File: quick_sort/quick_sort.py (hunk 3 above)
Comment (Naman9639) [inline quick_sort/quick_sort.py]: @t2013anurag Please review the changes.

**Comment by Naman9639** [comment]
Okay I am on it.

**Pull request merged**